### ISSUE TITLE
Disable max number of arguments Ruff Linting Rule [PLR0913]

### ISSUE THREAD

**Issue by AntonioMrtz** [AntonioMrtz/SpotifyElectron#155]
## Description
Disable backend linting rule that affects max number of arguments in backend functions. More info on [ruff PLR0913 rule](https://docs.astral.sh/ruff/rules/too-many-arguments/)

## Context

Rule is too strict and doesnt apply correctly to our use case. This forces us to disable the rule in some functions.

## How to do it

- Exclude rule `PLR0913` in `ruff.toml` in section:

`ignore = ["PLR0402","PLR0911"]`

- Delete `PLR0913` no qa comments in the code base

## Testing

Run Backend tests to check if everything is working correctly

---
__🖐️ Learn [how to CONTRIBUTE](https://antoniomrtz.github.io/SpotifyElectron_Web/docs/CONTRIBUTING/), all help is welcomed :)__

__📚 Check out the [project documentation](https://antoniomrtz.github.io/SpotifyElectron_Web/docs/) for more info__

__⭐ Help this project grow by giving it a star if you like it!__

__😵 Trello links are used for traceability, do not ask to join the board before leaving a comment in the related issue__

**Comment by AntonioMrtz** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Disable rule PLR0913 too many function arguments](https://trello.com/c/6wviMNoa/419-disable-rule-plr0913-too-many-function-arguments)

**Comment by berkaykrc** [comment]
May I be assigned to this issue please?

**Comment by AntonioMrtz** [comment]
Hi @berkaykrc , welcome abroad 🫡. Let me know if you have any questions 😁

**Comment by berkaykrc** [comment]
Hi @AntonioMrtz, thanks for welcoming me! Yes, I would like to get some help. I finished the task but cannot commit because .pre-commit giving error. I asked copilot about log file created from npm it's saying that the error caused by wsl2 which I use on Windows.

It's coming from prettier pre-commit hook since I run all hooks individually, only prettier caused that error.
> "The `pre_commit.util.CalledProcessError` indicates that the `npm pack` command is failing due to an `ERR_INVALID_URL` error. This is likely because the `npm` command is being executed in an environment where the paths are not correctly configured for WSL."

What would you like to see for error? I can provide terminal output or log files. I had locale error before that also. Fixed it by setting 
`sudo locale-gen en_US.UTF-8
sudo update-locale LANG=en_US.UTF-8`

**Comment by AntonioMrtz** [comment]
Hi @berkaykrc . Let's check a few things:

* `node` and `npm` should be installed in WSL. Looking at the logs it seems it's using your host device `node` and `npm`.
* With `node` version 20/22 and `npm` version 10 everything runs correctly. Are you using similar versions? Try `nvm` for installing a concrete node version. You can check it [here](https://github.com/nvm-sh/nvm)

If the error persist and you cannot fix it we can simply rely on the pipelines. The checks should be the same. :)

**Comment by berkaykrc** [comment]
Thank you, while I was waiting for your answer, I did some research on Stack Overflow and found some suggestions as you did. After the initial setup, I was displeased as the process took a considerable amount of time for me. Subsequently, with that combined took longer than actual coding.

**Comment by berkaykrc** [comment]
Hi @AntonioMrtz 
I'm curious, shouldn't this issue be closed?

**Comment by AntonioMrtz** [comment]
Yeah. It should, mabe issue number wasn't link correctly with the PR
